src/block/block_compact.c

```diff
@@ -211,17 +211,10 @@ __block_compact_estimate_remaining_work(WT_SESSION_IMPL *session, WT_BLOCK *bloc
     WT_EXT *ext;
     wt_off_t avg_block_size, avg_internal_block_size, depth1_subtree_size, leaves_per_internal_page;
     wt_off_t compact_start_off, extra_space, file_size, last, off, rewrite_size, size, write_off;
-    uint64_t n, pages_to_move, pages_to_move_orig, total_pages_to_move;
+    uint64_t n, pages_to_move, total_pages_to_move;
     int compact_pct_tenths, iteration;
     bool skip;
 
-    /*
-     * We must have reviewed at least some interesting number of pages for any estimates below to be
-     * worthwhile.
-     */
-    if (block->compact_pages_reviewed < WT_THOUSAND)
```

Comment:
This check has been moved outside the function.

Reply:
Moving this check outside of the function makes sense. Please add a comment here to document the assumption that enough pages have been reviewed for the statistics to be meaningful.

Reply:
Done in 0fb69dccfe940d06d89736735ea1205accd5d1c4